motionplan/motionPlanner.go

```diff
@@ -38,71 +38,32 @@ type motionPlanner interface {
 
 type plannerConstructor func(frame.Frame, *rand.Rand, golog.Logger, *plannerOptions) (motionPlanner, error)
 
-// PlanMotion plans a motion to destination for a given frame. It takes a given frame system, wraps it with a SolvableFS, and solves.
-func PlanMotion(ctx context.Context,
-	logger golog.Logger,
-	dst *frame.PoseInFrame,
-	f frame.Frame,
-	seedMap map[string][]frame.Input,
-	fs frame.FrameSystem,
-	worldState *frame.WorldState,
-	constraintSpec *pb.Constraints,
-	planningOpts map[string]interface{},
-) ([]map[string][]frame.Input, error) {
-	return motionPlanInternal(ctx, logger, dst, f, seedMap, fs, worldState, constraintSpec, planningOpts)
+type PlanRequest struct {
+	Logger          golog.Logger
+	Goal            *frame.PoseInFrame
+	Frame           frame.Frame
+	FrameSystem     frame.FrameSystem
+	Inputs          map[string][]frame.Input
```

Comment:
I think this could be more descriptive. StartConfiguration perhaps?

Reply:
I like it